example/src/index.js

```diff
@@ -147,6 +147,11 @@ class App extends React.Component {
           <h3>Custom input : Format credit card number</h3>
           <NumberFormat customInput={TextField} format="#### #### #### ####" />
         </div>
+
+        <div className="example">
```

Comment:
Can you also update doc (README.md) and types with these new props?
https://github.com/s-yadav/react-number-format/blob/master/typings/number_format.d.ts

Reply:
done in 
891393a
8270cba
respectively